### PR TITLE
Fix unallowed block error.

### PR DESCRIPTION
**Pull request by maciej-ka**
Displays error message 'Mixin "foo" does not accept a content block.'
for this example:

``` scss
@mixin foo() {}
@include foo() {}
```

Before, example was rendered without error and only failed if block
inside `@include` had some statements.

This is fix to:
https://github.com/sass/sass/issues/2106
Tests: sass/sass-spec#897

### CODE REVIEW

File: test/sass/engine_test.rb [hunk 1]
```diff
@@ -894,7 +900,7 @@ def test_properties_with_space_after_colon
 CSS
 foo
   bar : baz
-  bizz	: bap
+  bizz : bap
```
Comment (maciej-ka) [inline test/sass/engine_test.rb:903]: I don't know why diff marks this as a change, I haven't touched this line :/

**Comment by chriseppstein** [comment]
This looks good. We'll also want to add a corresponding spec to sass-spec.

**Comment by maciej-ka** [comment]
@chriseppstein sure. It will be in `spec/sass_3_5` untill end of this week.

**Comment by lokson** [comment]
@chriseppstein, @xzyfer https://github.com/sass/sass-spec/pull/896

**Comment by nex3** [comment]
@lokson Sorry for the long turnaround time on this! Can you rebase this onto the latest `master`? It has a change that'll make this run specs against your sass-spec pull request.

**Comment by maciej-ka** [comment]
@nex3 rebase onto master done.

**Comment by maciej-ka** [comment]
@nex3 https://github.com/sass/sass-spec/pull/897 rebased onto master

File: test/sass/engine_test.rb (hunk 1 above)
Comment (tagliala) [reply test/sass/engine_test.rb:903]: the original file has a tabulation after `bizz`, the edited file has a space. Maybe the text editor converted that tabulation automatically?

**Pull request closed without merging**